### PR TITLE
[tests-only][full-ci] reorganize test folders within the acceptance directory

### PR DESCRIPTION
**Pull request by prashant-gurung899**
## Description
This PR reorganizes the test folders within the acceptance directory.
   - move `TestHelpers` folder into `acceptance` folder.
   - move `bootstrap` folder out of `features` and into `acceptance` folder.

## Related Issue

- Closes: https://github.com/owncloud/ocis/issues/9872

## How Has This Been Tested?

-

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/bootstrap/bootstrap.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 $classLoader = new ClassLoader();
 
-$classLoader->addPsr4("TestHelpers\\", __DIR__ . "/../../../TestHelpers", true);
+$classLoader->addPsr4("TestHelpers\\", __DIR__ . "/../../../tests/acceptance/TestHelpers", true);
```
Comment (saw-jan) [inline tests/acceptance/bootstrap/bootstrap.php:28]:
```suggestion
$classLoader->addPsr4("TestHelpers\\", __DIR__ . "/../TestHelpers", true);
```

File: tests/parallelDeployAcceptance/features/bootstrap/bootstrap.php [hunk 2]
```diff
@@ -25,9 +25,9 @@
 $classLoader = new ClassLoader();
 $classLoader->addPsr4(
 	"",
-	__DIR__ . "/../../../tests/acceptance/features/bootstrap",
+	__DIR__ . "/../../../tests/acceptance/bootstrap",
 	true
 );
 
-$classLoader->addPsr4("TestHelpers\\", __DIR__ . "/../../../TestHelpers", true);
+$classLoader->addPsr4("TestHelpers\\", __DIR__ . "/../../../acceptance/TestHelpers", true);
```
Comment (saw-jan) [inline tests/parallelDeployAcceptance/features/bootstrap/bootstrap.php:32]: you can leave this file as it is. no changes required.

Comment (prashant-gurung899) [reply tests/parallelDeployAcceptance/features/bootstrap/bootstrap.php:32]: Okay. Makes sense as we are deleting those anyway.

**Review by saw-jan — APPROVED** [review]
LGTM :+1:

**Pull request merged**